pkg/registry/batch/job/strategy_test.go

```diff
@@ -88,8 +94,9 @@ func testJobStrategy(t *testing.T) {
 		},
 		Spec: batch.JobSpec{
 			Selector:                validSelector,
+			Suspend:                 newBool(true),
```

Comment:
we should get rid of these methods and use the `pointer` package. But we can do that as followup for the entire file

Reply:
Changed it everywhere